src/uniprotkb/components/query-builder/AdvancedSearchContainer.tsx

```diff
@@ -56,8 +55,10 @@ type State = {
 export class Search extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    const { queryString } = props;
-    this.state = { queryString };
+
+    // const { queryString } = props;
```

Comment:
Do we want to keep these commented out lines?

Reply:
I wasn't sure what the component was doing, and it seemed functional after commenting big chunks of that file. I assumed it was leftover code from a while ago, but I mainly commented it to make TypeScript API but keep the code handy in case something was really necessary.

Reply:
Ok - just merge it and I'll have a look